tox.ini

```diff
@@ -36,75 +31,21 @@ whitelist_externals =
     env
 
 [testenv:server]
-description = Runs all non-Python3-based server unit/functional tests
+description = Runs all server unit/functional tests
 passenv = PBENCH_UNITTEST_SERVER_MODE
 deps =
     -r{toxinidir}/server/requirements.txt
     -r{toxinidir}/server/test-requirements.txt
 commands =
-    bash -c "./server/bin/unittests {posargs}"
+    bash -c "./server/exec-unittests {envdir} {posargs}"
```

Comment:
You like invoking `exec-unittests` on a relative path instead of on `{toxinidir}/server/exec-unittests` (like line 38...or does tox not expand the references inside double quotes?)?

Ditto line 48.

Reply:
Did I make a change to invoke on a relative path?  It seems like that it how it was before.  I believe the commands are always run in the directory in which the `tox.ini` file is found.

Reply:
As one can see from the diff's above, your change did not introduce the relative path; however, as a reviewer, I find it jarring that we use a relative path at line 40 when we use what seems like an absolute path (or a potentially _different_ relative path) at line 38 (et al.).  I'm concerned that inconsistencies like these open cracks which can admit surprising results.  (I'm aware of the power of position-independent code, but I'm also aware of the destructiveness of misplaced assumptions.)